Common.mk

```diff
@@ -1,6 +1,6 @@
 # Disable built-in rules and variables
 MAKEFLAGS+=--no-builtin-rules --warn-undefined-variables
-SHELL=/bin/bash
+SHELL=bash
```

Comment:
After updating my mac to monterey, make started using the mac default bash at /bin/bash which doesn't support `declare -A` which we use in various scripts.  I have bash 5 installed via brew.  I am not sure if at one point before the upgrade i had bin/bash symlinked differently or what.  I *think* this change should be ok and still work in the al2 builder base, but well see in prow.